app/configuration/Config.scala

```diff
@@ -68,7 +68,7 @@ object Config {
   object Zuora {
     private val stageConfig = config.getConfig("touchpoint.backend.environments").getConfig(stage)
 
-    val paymentDelay = Days.days(stageConfig.getInt("zuora.paymentDelayInDays"))
+    val defaultDigitalPackFreeTrialPeriod = Days.days(stageConfig.getInt("zuora.paymentDelayInDays"))
```

Comment:
I didn't like the name collision of the Digital Pack free trial period length and the payment delay when orchestrating a subscription.